x-pack/plugins/observability/public/components/shared/exploratory_view/series_builder/columns/report_types_col.tsx

```diff
@@ -7,27 +7,33 @@
 
 import React from 'react';
 import { i18n } from '@kbn/i18n';
+import { map } from 'lodash';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
 import styled from 'styled-components';
-import { ReportViewTypeId, SeriesUrl } from '../../types';
+import { ReportViewType, SeriesUrl } from '../../types';
 import { useSeriesStorage } from '../../hooks/use_series_storage';
 import { DEFAULT_TIME } from '../../configurations/constants';
 import { useAppIndexPatternContext } from '../../hooks/use_app_index_pattern';
+import { ReportTypeItem, SELECT_DATA_TYPE } from '../series_builder';
 
 interface Props {
   seriesId: string;
-  reportTypes: Array<{ id: ReportViewTypeId; label: string }>;
+  reportTypes: ReportTypeItem[];
 }
 
 export function ReportTypesCol({ seriesId, reportTypes }: Props) {
-  const { setSeries, getSeries } = useSeriesStorage();
+  const { setSeries, getSeries, firstSeries, firstSeriesId } = useSeriesStorage();
 
   const { reportType: selectedReportType, ...restSeries } = getSeries(seriesId);
```

Comment:
It's unclear to me why reportType was pulled off the series object only. Is it because it's used for comparisons often? If so, it seems like dataType could also be destructured, as it's used quiet often